lib/Module/Starter/Smart.pm

```diff
@@ -170,11 +170,12 @@ sub create_t {
 }
 
 sub _create_t {
-    my $self = shift;
+    my $self     = shift;
+    my $testdir  = @_ == 2 ? 't' : shift;
```

Comment:
Thanks for the pull request. This line will ignore the 2nd arg (and use 't' instead) when only two args are given. Is this to be expected with the new protocol?